dtfs-central-api/api-tests/v1/deals/get-deal.api-test.js

```diff
@@ -0,0 +1,89 @@
+const { generateParsedMockPortalUserAuditDatabaseRecord } = require('@ukef/dtfs2-common/change-stream/test-helpers');
```

Comment:
The tests in this folder have been split from the existing test files and not refactored.